webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
- [ ] OPTIONAL
_Mode_ option is not set for _webpack_. Kindly set the `mode` and either assign its value to `development` or `production` within the _module.exports_

**Webpack config error:**
![image](https://user-images.githubusercontent.com/12745474/134145548-88d2d895-844b-466b-88a0-27c7b27a37a5.png)